docs/install/installEverest.md

```diff
@@ -0,0 +1,105 @@
+
+## Prerequisites
+
+Before getting started with Percona Everest, we recommend that you:
+
+1. Install [Docker](https://docs.docker.com/engine/install/) and its compose plugin (included by default in the aforementioned docker engine installation instructions).
+2. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) for EKS or the [gcloud CLI](https://cloud.google.com/sdk/docs/install) for GKE.
+3. Download the kubeconfig file from the Kubernetes cluster that you want to use with Everest. By default, Everest uses the file available under *~/.kube/config.*
+
+## Get started
+
+To install and provision Percona Everest:
+
+1. Download the Docker compose file:
+```
+curl https://raw.githubusercontent.com/percona/percona-everest-backend/v0.3.0/deploy/quickstart-compose.yml -o quickstart-compose.yml
+```
+2. Initialize the Everest container and its internal PostgreSQL database by executing one of the following commands, each offering distinct access configurations:
+
+    - to limit access to the localhost interface (default):
+    ```sh
+    docker compose -f quickstart-compose.yml up -d
+    ```
+    - to expose Everest though a specific interface, use that interface's IP address: 
+    ```sh
+    EVEREST_BIND_ADDR=<ip address>; docker compose -f quickstart-compose.yml up -d
+    ```
+    - to allow access from any interface on the host machine (not recommended):
+    ```sh
+    EVEREST_BIND_ADDR=0.0.0.0; docker compose -f quickstart-compose.yml up -d
+    ```
+  
+3. (Optional) Verify if the services started correctly:
+```sh 
+docker compose -f quickstart-compose.yml ps --services --filter 'status=running'
+```
+
+    ??? example "Expected output"
+        ```
+            everest
+            pg
+        ```
+
+4. Download the latest release of the [everestctl](https://github.com/percona/percona-everest-cli/releases) command to provision Percona Everest.
+5. Rename the downloaded file using the following command and replacing the placeholder `everestctl-darwin-amd64` to match the file downloaded in the previous step: 
+    ```sh
+    mv everestctl-darwin-amd64 everestctl
+    ```
+6. Modify the file permissions: 
+    ```sh
+    chmod +x everestctl
+    ```
+7. From the installation wizard, provision and register the Kubernetes cluster in Everest using the following command. Everest will search for the kubeconfig file in the `~/.kube/config` path. If your file is located elsewhere, add the `KUBECONFIG` environment variable before running the command.
+
+    !!! note alert alert-primary "Note"
+       - If you don't enable monitoring during this provisioning step then you won't be able to enable it from the UI later. Make sure to fill in the monitoring details in the wizard.
+       - If you are using a PMM server instance with a self-signed certificate you cannot use HTTPS in the PMM URL endpoint.
+
+    ```sh
+    KUBECONFIG=~/.kube/config; ./everestctl install operators
```

Comment:
.kube/config it is not a custom path :) everestctl will use it by default without KUBECONFIG var, I think we can remove it at all
 ```suggestion
   ./everestctl install operators
```